content/blog/_posts/inclusive-infra.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Towards More Inclusive Infrastructure"
+date: 2021-04-14
+slug: inclusive-infra
+authors: Reinhardt Quelle
+---
+Posted by Reinhardt Quelle, Dir. Site Reliability Engineering for Cisco Emerging Technolgies and Incubation
+
+This post will be short, because it really is _that simple_.
+
+My team builds platform services that underlie the multiple projects within the Emerging Technology and Incubation team.  Our mission is to build 'paved roads' for developers; we create common reusable patterns and services that any engineer or venture needs to deliver an application.
+
+Recently, one of my peers pinged me in chat and pointed out that someone on my team had just announced the expansion of our CI build environment with the addition of "slaves 10-15". She inquired about the use of the term “slave” and I immediately knew she was pointing to a form of microaggression in code and something not in line with our values.
+
+
+She is right.  It's not only inappropriate to perpetuate the use of terms that are harmful to our diverse employees; it's also specifically against our company [policy](https://www.cisco.com/c/en/us/about/social-justice/inclusive-language-policy.html) to do so. 
+
+Since these hosts had been created via "infrastructure as code", it was a simple matter to send a pull request to the repo containing that code with more inclusive - and indeed more descriptive - names, and re-deploy. 
```

Comment:
```suggestion
Since these hosts had been created using an infrastructure as code approach, all they had to do was send me a pull request to the repo containing that code with more inclusive - and indeed more descriptive - names, and re-deploy. 
```

Reply:
Questions: 
1) What more inclusive names did you use in this case?
2) Who decided what was more "inclusive"?

Reply:
Leaving as is, as specifics of the pull request are less relevant.  Early draft had more details (including a nerdy ```sed 's/slave/node'  build-hosts.tf```, but my earlier editors thought it detracted from the "keep it simple" and the tools (and in fact new terms) were very environment-specific and not necessary.